actions/plugins/publish/change-plugin-scope/action.yml

```
@@ -0,0 +1,52 @@
+name: Change plugin scope
+
+inputs:
+  gcom-token:
+    description: GCOM token with permissions to update a plugin version.
+    required: true
+  gcloud-auth-token:
+    description: |
+      Google Cloud auth token for IAP access.
+      Required only for to dev or ops access.
+    required: false
+    default: ""
+  environment:
+    description: |
+      Environment where the plugin is published.
+      Can be 'dev', 'ops', or 'prod'.
+    required: true
+  scopes:
+    description: |
+      Scope to promote the plugin to.
```

```suggestion
      Scope to apply to the plugin.
```
Let's not use the word "promote" (here and any other place in this workflow) we are not really promoting plugins here. Generally speaking this workflow let you change a plugin scope.